app.py

```diff
@@ -0,0 +1,44 @@
+# app.py
+from urllib.parse import urlparse, urlunparse
+from flask import Flask, redirect, jsonify, render_template, url_for, request, abort, Response
+from requests import get
+
+import json
+import requests
+
+app = Flask(__name__)
+
+# Web Pages
+
+SITE_NAME = "http://wallet.v.systems:9922"
```

Comment:
Is there must use `http` with port 9922? Is it better to change to `https://wallet.v.systems/api`? Because port 9922 would not ensure be opened for all services.

Reply:
this is just an example. SITE_NAME need to be adjusted by user

Reply:
SITE_NAME best be called NODE_API_URL